### DISCUSSION TITLE
Give alternatives answer for Q "How wide is the road here?"

### DISCUSSION THREAD

**Discussion by hoizan** [streetcomplete/StreetComplete#4996] [Feedback]
Instead of updating the width of a road in measurement unit like in meters, do  cater for  alternative answer in number of car/truck/motorcycle lanes etc . 
This is mush easier  for road surveying. I think hardly anyone  will physically measure the width of a road.

**Comment by Helium314** [comment]
You can just disable this quest and only answer the lanes quest.

**Comment by hoizan** [comment]
Yes. I understand they are independent tasks for different situations. I am
thinking for the specific task of checking the width of a road, it is
easier to fulfill by indicating by typical width of a car, truck, etc

I I'll just disable the width task then

Thanks.
